packages/block-editor/src/components/background-image-control/style.scss

```diff
@@ -23,7 +23,10 @@
 
 	.components-dropdown {
 		display: block;
-		height: 36px;
+
+		.block-editor-global-styles-background-panel__dropdown-toggle {
+			height: 36px;
```

Comment:
```suggestion
			height: 40px;
```

Reply:
Thanks for the review @mirka. I've updated the height in the latest commit.

![Screenshot 2024-12-30 at 11 48 30 AM](https://github.com/user-attachments/assets/8328390b-299a-42e9-a292-445f5c1886c4)